homework-classes/RepoView.js

```diff
@@ -20,6 +20,27 @@
      */
     render(repo) {
       // TODO: replace this comment and the console.log with your own code
+      this.container.textContent = '';
+      const ul = createAndAppend('ul', this.container);
+      const li = createAndAppend('li', ul);
+      const table = createAndAppend('table', li);
+      const titles = ['Repository:', 'Description:', 'Forks:', 'Updated:'];
+      const keys = ['name', 'description', 'forks', 'updated_at'];
+
+      titles.forEach((title, index) =>{
+      	  let tr = createAndAppend('tr', table);
+          createAndAppend('th', tr, { text: title });
+          if (index > 0) createAndAppend('td', tr, { text: repo[keys[index]] });
+          else {
+            const td = createAndAppend('td', tr);
+            createAndAppend('a', td, {
+              href: repo.html_url,
+              text: repo.name,
+              target: '_blank',
+            });
+          }
```

Comment:
early return will work too for legibility, trust me you may not see the difference now, but you will appreciate that someday 🙂 
```suggestion
          if (index > 0) {
            createAndAppend('td', tr, { text: repo[keys[index]] });
            return;
          }

            const td = createAndAppend('td', tr);
            createAndAppend('a', td, {
              href: repo.html_url,
              text: repo.name,
              target: '_blank',
            });
```

Reply:
I trust you :) I will keep this in mind.